indigo_app/templates/place/detail.html

```diff
@@ -17,7 +17,9 @@ <h5 class="mb-0">Types of works</h5>
           {% for item in subtypes %}
             {% if item.1 %}          
               <div class="row mb-2">
-                <div class="col-4">{{ item.0 }}</div>
+                <div class="col-4">
+                  <a class="text-dark" href="{% url 'place_works' place=place.place_code %}?subtype={{ forloop.counter }}">{{ item.0 }}</a>
```

Comment:
`?subtype={{ forloop.counter }}` isn't giving the right subtype. first, 'Act' isn't a subtype. and there are subtypes that have 0 works in a place. plus the works and subtypes are ordered differently. so it'll probably be better to include the subtype's `id` in `get_works_by_subtype`, but you'll still have to deal with 'Act' separately. worth noting that there isn't currently a mechanism for filtering on `Act` in the work listing view – maybe that needs to be addressed first? (I'll make an issue for it)

Reply:
https://github.com/laws-africa/indigo/issues/888

Reply:
I used the same ordering as in on the Metrics section as can be seen here:
![image](https://user-images.githubusercontent.com/8082197/71073373-4f24ae80-2191-11ea-8627-3d2f7bf5a995.png)

And yes, I think a mechanism for filtering on acts is required.

Reply:
I don't disagree with your ordering of the number of works in the view, but `subtype=1`, `subtype=2`, etc don't match up to `Proclamation`, `Government Notice`, etc.

e.g. for my db: 
![image](https://user-images.githubusercontent.com/32566441/71087422-cd3e8080-21a4-11ea-9b35-17c7ba96a5cb.png)

Clicking on `90` for `Proclamation` takes me to http://127.0.0.1:8000/places/na/works/?subtype=2, which is filtered to `Statutory Instrument`:
![image](https://user-images.githubusercontent.com/32566441/71087479-f65f1100-21a4-11ea-9192-9a64c1fa72cf.png)

(my subtypes and their ids are:
By-law (by-law): 1
Statutory Instrument (si): 2
Government Notice (gn): 3
Proclamation (p): 4
ord (ord): 5)

I think that if you include the `id` of the subtype when you make this list at around https://github.com/laws-africa/indigo/blob/master/indigo_app/views/places.py#L178, filtering by `?subtype={{item.3}}` in the template should work.